.github/workflows/build_release.yaml

```diff
@@ -77,17 +77,20 @@ jobs:
       - name: Build app for production
         run: npm ci --only=production --prefix services/webapi/
 
-      - name: Package application
+      - name: Package application without node_modules
         run: npm run package --prefix services/webapi
 
+        # Currently needed as ORYX builds are not working on Arc
+      - name: Package application with node_modules for Arc deployment
```

Comment:
Does this run for non-arc builds as well?  Any side-effects on running this in normal mode if so?

Reply:
We're producing two artifacts for the app, one without node_modules, and one with. The one is needed for arc, the other can be used in Azure so that modules will be compiled on the platform where it runs. The impact of always building both is storage taken up in GitHub artifacts. We could look into doing this conditionally if you would like that?